src/buildblock/ProjDataFromStream.cxx

```diff
@@ -376,6 +417,93 @@ ProjDataFromStream::set_viewgram(const Viewgram<float>& v)
 }
 
 
+std::vector<std::streamoff>
+ProjDataFromStream::get_offsets_bin(const int segment_num,
+                                    const int ax_pos_num,
+                                    const int view_num,
+                                    const int tang_pos_num) const
+{
```

Comment:
would be better to pass a `Bin` as argument. also immediatelly TOF ready

Reply:
👍